gpkit/keydict.py

```diff
@@ -5,24 +5,24 @@
 
 
 class KeyDict(dict):
-    """KeyDicts allow storing and accessing the same value with multiple keys
```

Comment:
For what it's worth I thought this first sentence was a great intro; I'd vote to keep it.